client/src/components/geneExpression/gene.tsx

```diff
@@ -308,6 +306,6 @@ function mapStateToProps(state: RootState, ownProps: OwnProps): StateProps {
       state.colors.colorMode !== "color by categorical metadata",
     isScatterplotXXaccessor: state.controls.scatterplotXXaccessor === gene,
     isScatterplotYYaccessor: state.controls.scatterplotYYaccessor === gene,
-    isGeneInfo: state.controls.gene === gene,
+    isGeneInfo: state.controls.geneInfo.gene === gene,
```

Comment:
sanity check: do we need any optional chaining here or are we super sure the chained getters will always work?

E.g.,

```ts
state.controls.geneInfo?.gene
```

Reply:
I think since the state object is initialized with default values, it will always work?

`
    geneInfo: {
      gene: null,
      geneName: "",
      geneSummary: "",
      geneSynonyms: [""],
      geneUrl: "",
      showWarningBanner: false,
      infoError: null,
      loading: false,
    },
`

Reply:
Oh yes you're right! Sorry I missed that 😆